neurom/fst/sectionfunc.py

```diff
@@ -123,3 +123,10 @@ def strahler_order(section):
             return max_so_children + 1
         return max_so_children
     return 1
+
+
+def locate_segment_position(section, fraction):
```

Comment:
section id isn't something we have tried to make constant/public, and doesn't necessarily match what other software expects the section id to be.  I'm hesitant to have this exposed, unless it's understood that the segment id is only stable during one instance of a morphology, and that guarantees aren't made between other invocations/versions/etc

does that make sense?

Reply:
Whether we want it or not, section IDs are _already_ used for accessing geometry through NeuroM.
I understand that it is problematic, but saying "we give no guarantees" is hardly an option.

Reply:
That is quite useful for the morphology analysis as well. Thanks for the implementation!

Reply:
Glad it found reuse so soon :); but please note that Mike is right: section / segment ID semantics is not well defined yet.

Reply:
Yes, I am aware of that. The need for this was primarily internal so I assume it is consistent for the same morphology internally in NeuroM which works for me at the moment :)